src/iop/basecurve.c

```diff
@@ -1792,22 +1792,15 @@ static gboolean _scrolled(GtkWidget *widget, GdkEventScroll *event, gpointer use
 
   if(c->selected < 0) return TRUE;
 
-  int handled = 0;
-  float dy = 0.0f;
-  if(event->direction == GDK_SCROLL_UP)
-  {
-    handled = 1;
-    dy = BASECURVE_DEFAULT_STEP;
-  }
-  if(event->direction == GDK_SCROLL_DOWN)
+  gdouble delta_x, delta_y;
```

Comment:
I went ahead and added a commit to make all scroll events be for base/tonecurve be handled the same, whether mouse scrollwheel or touch etc. This makes the code simpler, but does change behavior from what is in git master. But this is a more sensible behavior for scrollwheel mice as well as the other cases?